_presenters/julia-solorzano.md

```diff
@@ -10,14 +10,16 @@ twitter: null
 website: null
 ---
 
-Julia Solórzano is the Security Experience Supervisor for [Login.gov](https://login.gov/) and is based in Durham, NC. She leads security, user experience and engineering teams, and coaches people to help them reach their full potential to create amazing things. Outside of Login.gov, her most recent work includes the [Department of Veterans API](https://developer.va.gov/) program and the [U.S. Web Design System](https://designsystem.digital.gov/). She is also a professor at Georgetown University where she teaches technology design and development.
+Julia Solórzano is the Security Experience Supervisor is based in Durham, NC. She leads security, user experience and engineering teams, and coaches people to help them reach their full potential to create amazing things. Outside of Login.gov, her most recent work includes [Login.gov](https://login.gov),  [Department of Veterans API](https://developer.va.gov/) program and the [U.S. Web Design System](https://designsystem.digital.gov/). She is also an adjunct professor at Georgetown University where she teaches technology design and development.
```

Comment:
It still reads as "Outside of Login.gov, her most recent work includes Login.gov." 

@juliaelman do you want to rephrase this?